tools/testbench/topology.c

```diff
@@ -339,8 +341,12 @@ static int load_filewrite(struct sof *sof, int comp_id, int pipeline_id,
 			output_file_index);
 		return -EINVAL;
 	}
+	printf("debug: filewrite.fn: %s\n", tp->output_file[output_file_index]);
 	filewrite.fn = strdup(tp->output_file[output_file_index]);
-	tp->fw_id = comp_id;
+	if (output_file_index == 0) {
+		tp->fw_id = comp_id;
+	}
+	output_file_index++;
```

Comment:
Load filewrite with different output file name to each DAI.
argument: -o output1.raw,output2.raw